README.md

```diff
@@ -61,9 +61,14 @@
 - Press `Ctrl + Alt + D` to check the price of the item\
   *Opens a window, but does not hide it automatically when moving the mouse*
 - Press `F5` to go to hideout
+- Press `F9` to exit to character selection
 - `Ctrl + MouseWheel` to scroll through stash tabs\
   *You can also disable "Mousewheel Zoom" in PoE settings (Options -> UI -> 3-rd check mark starting from the bottom)*
 - `Alt + W` to open the item on Wiki
+- `Crtl + Shift + Space` to open overlay
```

Comment:
This seems to be just Shift + Space (no Ctrl).

Reply:
That sounds either like a bug, or the default changed. What does it say in the Hotkeys settings?

Reply:
Looks like it's been that way a while: https://github.com/SnosMe/awakened-poe-trade/blob/master/src/ipc/types.ts#L66

Reply:
So it is, I've fixed the PR.

I think I'd changed my setting as I was hitting it accidentally,